websites/T/TikTok/presence.ts

```diff
@@ -90,12 +103,14 @@ presence.on("UpdateData", async () => {
 				}
 			];
 		} else {
-			presenceData.details = (await strings).viewProfile;
-			presenceData.state = `${document
-				.querySelector(".share-sub-title")
-				?.textContent.trim()} (@${document
-				.querySelector(".share-title")
-				?.textContent.trim()})`;
+			presenceData.details = await strings.viewProfile;
+			presenceData.state = `@${
+				document.querySelector("[data-e2e=user-title]").textContent
+			} (${document.querySelector("[data-e2e=user-subtitle]").textContent})`;
+
+			presenceData.largeImageKey = document.querySelector<HTMLMetaElement>(
+				'meta[property="og:image"]'
+			).content;
 			presenceData.buttons = [
 				{
 					label: (await strings).buttonViewProfile,
```

Comment:
You should migrate to use the strings type like in Netflix presence etc

Reply:
Will check netflix presence’s code out

Reply:
It is already look how the strings are typed

Reply:
so huh wdym then

Reply:
Use the type here, this removes the need of `(await strings).` constantly
https://github.com/PreMiD/Presences/pull/5833/files/84f76ed9d1d61d4577739648bdfae228b99f0ac9#diff-564cbd292c5b92c85f3072d482a5c5cb476da7ac3e930559ec99470f96c2b7e1R16